autoscraper/auto_scraper.py

```diff
@@ -67,24 +66,22 @@ def _child_has_text(self, child, text):
                     child.wanted_attr = key
                     child.is_full_url = True
                     return True
+
         return False
 
     def _get_children(self, soup, text):
-        text = text.strip().rstrip()
+        text = text.strip()
         children = reversed(soup.findChildren())
         children = list(
             filter(lambda x: self._child_has_text(x, text), children))
+
         return children
 
-    @staticmethod
-    def unique(item_list):
-        unique_list = []
-        for item in item_list:
-            if item not in unique_list:
-                unique_list.append(item)
-        return unique_list
+    @classmethod
+    def unique(cls, item_list):
+        return list(set(item_list))
```

Comment:
It doesn't preserve the ordering of the list. Also items may not be hashable.
No need for `cls` here.

Reply:
it's a general utility function. it shouldn't be part of the class as it's not relevant/bounded to it.